indexer/migrations/20230523_create_schema.sql

```diff
@@ -0,0 +1,99 @@
+/**
+ * BLOCK DATA
+ */
+
+CREATE TABLE IF NOT EXISTS l1_block_headers (
+	hash        VARCHAR NOT NULL PRIMARY KEY,
+	parent_hash VARCHAR NOT NULL,
+	number      NUMERIC NOT NULL,
+	timestamp   INTEGER NOT NULL,
+
+    CONSTRAINT
+        l1_parent_hash_fkey FOREIGN KEY(parent_hash) REFERENCES l1_block_headers(hash)
+);
+
+CREATE TABLE IF NOT EXISTS l2_block_headers (
+    -- Block header
+	hash                     VARCHAR NOT NULL PRIMARY KEY,
+	parent_hash              VARCHAR NOT NULL,
+	number                   NUMERIC NOT NULL,
```

Comment:
The old schema had a unique constraint on this

Reply:
yea i saw those constraints. Some of them didn't make sense

Good to add an index on the block number though

Reply:
for the extra constraints, lets add them as needed